lisa/tools/aria2.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+
+class Aria2c(Tool):
+    @property
+    def command(self) -> str:
+        return "aria2c"
+
+    def can_install(self) -> bool:
+        return True
+
+    def _install(self) -> bool:
+        assert isinstance(self.node.os, Posix)
+        self.node.os.install_packages("aria2")
+        return self._check_exists()
+
+    def download_file(self, url: str, file_name: str) -> None:
```

Comment:
Is it used to download file? How about wget?

Reply:
It is used for downloading file. This is the default tool used in v2 probably because nested image file size can be large and this allows simultaneous downloads.

Reply:
ok. Change the method name to `download`.

Reply:
I moved to using Wget so this comment is not applicable now.

Reply:
Is the wget fast enough?

Reply:
It's a bit slower(~30sec extra time) than aria but seems to work fine. Another issue with aria is that it's not supported on sles, so we would have to use wget on sles.

Reply:
We can go with wget. But in general, if different distros have different tools, we can create a virtual tool like `service`. It can wrap different tools underlying, and provide the same interface.